nncf/onnx/graph/onnx_graph.py

```diff
@@ -343,56 +349,19 @@ def _get_tensor_shape(tensor: Union[onnx.ValueInfoProto, onnx.TensorProto]) -> L
                     return shape
         return shape
 
-    def get_edge_shape(self, edge_name: str) -> List[int]:
-        """
-        Returns a shape of the edge with the name 'edge_name'.
-        If the activations tensors were not filled in self._activations_tensor_name_to_value_info, it updates them.
-        If after updating of the self._activations_tensor_name_to_value_info, there is still no such tensor,
-        do shape inference of the model.
-
-        :param edge_name: The name of the edge.
-        :return: Shape of the tensor on that edge.
-        """
-        if self._activations_tensor_name_to_value_info is None:
-            self._update_activation_tensors()
-        if edge_name in self._activations_tensor_name_to_value_info:
-            return ONNXGraph._get_tensor_shape(self._activations_tensor_name_to_value_info[edge_name])
-        self._update_activation_tensors(do_shape_inference=True)
-        if edge_name in self._activations_tensor_name_to_value_info:
-            return ONNXGraph._get_tensor_shape(self._activations_tensor_name_to_value_info[edge_name])
-        raise RuntimeError('There is no edge with the name {}'.format(edge_name))
-
-    def get_edge_dtype(self, edge_name: str) -> int:
-        """
-        Returns the data type of the edge with the name 'edge_name'.
-        If the activations tensors were not filled in self._activations_tensor_name_to_value_info, it updates them.
-        If after updating of the self._activations_tensor_name_to_value_info, there is still no such tensor,
-        do shape inference of the model.
-
-        :param edge_name: The name of the edge.
-        :return: Shape of the tensor on that edge.
-        """
-        if self._activations_tensor_name_to_value_info is None:
-            self._update_activation_tensors()
-        if edge_name in self._activations_tensor_name_to_value_info:
-            if isinstance(self._activations_tensor_name_to_value_info[edge_name], onnx.ValueInfoProto):
-                return self._activations_tensor_name_to_value_info[edge_name].type.tensor_type.elem_type
-            return self._activations_tensor_name_to_value_info[edge_name].data_type
-        self._update_activation_tensors(do_shape_inference=True)
-        if edge_name in self._activations_tensor_name_to_value_info:
-            if isinstance(self._activations_tensor_name_to_value_info[edge_name], onnx.ValueInfoProto):
-                return self._activations_tensor_name_to_value_info[edge_name].type.tensor_type.elem_type
-            return self._activations_tensor_name_to_value_info[edge_name].data_type
-        raise RuntimeError('There is no edge with the name {}'.format(edge_name))
-
-    def get_edge_dtype_name(self, edge_name: str) -> str:
+    @staticmethod
+    def get_edge_dtype(edge: Union[onnx.ValueInfoProto, onnx.TensorProto]) -> np.dtype:
```

Comment:
The same comment about keeping it as `staticmethod`.
Also, this method is always followed by `onnx.helper.np_dtype_to_tensor_dtype` or `GraphConverter.convert_np_dtype_to_nncf_dtype` while returning `tensor_dtype_to_np_dtype`, for example. Can we avoid the number of methods for type-to-type conversion and use only the `ONNX` types and `NNCF` types?

Reply:
This should decrease the lines of code, I will change format from `numpy ` to `ONNX`

Reply:
Done